### PR TITLE
[Doc] Fix FAQ broken link

### PR DESCRIPTION
**Pull request by liketic**
The PIP for message deduplication is unaccessable and should PIP-6.

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -92,13 +92,13 @@ This is a lightweight option of having multiple clusters for different users, si
 There is regex subscription coming up in Pulsar 2.0. See [PIP-13](https://github.com/apache/pulsar/wiki/PIP-13:-Subscribe-to-topics-represented-by-regular-expressions).
 
 ### Does Pulsar have, or plan to have, a concept of log compaction where only the latest message with the same key will be kept ?
-Yes, see [PIP-9](https://github.com/ivankelly/pulsar-wiki/pull/1/files) for more details.
+Yes, see [PIP-6](https://github.com/apache/pulsar/wiki/PIP-6:-Guaranteed-Message-Deduplication) for more details.
```
Comment (sijie) [inline faq.md]: I don't think this is the right PIP. It should be PIP-14.

Comment (liketic) [reply faq.md]: @sijie Please review again.

**Comment by sijie** [comment]
run java8 tests

**Comment by liketic** [comment]
run java8 tests

**Comment by liketic** [comment]
run java8 tests

**Comment by sijie** [comment]
run java8 tests

**Comment by jiazhai** [comment]
run java8 tests

**Comment by jiazhai** [comment]
run integration tests

**Pull request merged**